index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <script
+      src="https://kit.fontawesome.com/71d90399a2.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
```

Comment:
Your design is nice, but I feel you can add, for instance, the right background color according to the Figma design you selected, we are going to check that you make almost an exact copy from the Figma design due to a lot of companies expect this ability from front-end developers.
- [ ] Try to add and fix more/some styles to make it more similar to the Figma design.
- [ ] Fix the menu logo size, make it smaller.
- [ ] [OPTIONAL] Add a little bit more padding for your body or main content to center a little bit your content 😉 

Figma design you selected  | Your design

![image](https://user-images.githubusercontent.com/60164239/133576893-4903b526-108a-4b93-8e83-1d8bd049a6b8.png)